packages/app-content-pages/src/screens/Publications/Publications.js

```diff
@@ -54,15 +53,16 @@ function Publications ({
   )
 
   const sidebar = (
-    <Box as='ul' gap='small'>
+    <Box as='ul' gap='small' name='Filter by category'>
```

Comment:
`gap` here is inserting invalid `div` elements inbetween the list items, resulting in this accessibility warning:
```
<ul> and <ol> must only directly contain <li>, <script> or <template> elements
```

This is a bug in Grommet, but we've worked around it before in https://github.com/zooniverse/front-end-monorepo/pull/1598.

Reply:
Is `name` a property for Grommet flexboxes? I thought it would be `aria-label`. Or, better yet, wrap this list in a labeled navigation element:

```html
<nav aria-label='Filter by category'>
  <ul>
    <li><a href='#space'>Space</a></li>
    <li><a href='#nature'>Nature</a></li>
  </ul>
</nav>
```

"Filter by category" should also be localised here, so that it can be translated into other languages.

Reply:
Good point about the localisation and `nav`, I'll refactor accordingly